internal/provider/resource_tfe_workspace.go

```diff
@@ -112,9 +116,29 @@ func resourceTFEWorkspace() *schema.Resource {
 
 			"auto_destroy_at": {
 				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
Unfortunately most of the raw config/state checking in this PR is due to this value being changed to `computed`. The main problem is insuring that unsetting `auto_destroy_at` (when `auto_destroy_activity_duration` is also unset) sends a null value to the TFC API.